src/aws/Dockerfile

```diff
@@ -118,6 +118,10 @@ RUN yum install -y \
     yum clean all  && \
     rm -rf /var/cache/yum
 
+## install clamav
+RUN yum -y install clamav && \
+    freshclam
```

Comment:
Once installed you should probably execute `freshclam` to update the database signature ? https://docs.clamav.net/manual/Usage/SignatureManagement.html

Reply:
Sorry I don't get it… `freshclam` is already executed…

Reply:
Yep sorry I though you were just installing it but in fact is it installed by clamav... and you execute it once clamav installed...